server/routes/service.ts

```diff
@@ -194,7 +194,9 @@ serviceRoutes.get<{ tmdbId: string }>(
         language: req.locale ?? (req.query.language as string),
       });
 
-      const response = await sonarr.getSeriesByTitle(tv.name);
+      const response = await sonarr.getSeriesByTitle(
+        tv.original_language === 'en' ? tv.original_name : tv.name
```

Comment:
This won't help unless the original language was English. Would be better to fetch the English name (when locale isn't English) and then perform the Sonarr search.